xhtml2pdf/util.py

```diff
@@ -46,6 +47,7 @@
     import urllib.request as urllib2
 except ImportError:
     import urllib2
+    import urllib2
```

Comment:
import urllib2  is twice import in this file

Reply:
Thanks for pointing out. I've fixed it.